the-collab-lab/tcl-62-smart-shopping-list
# Styling the home page

## Description

When a user visits the home page without being logged into an existing list, the navigation bar links 'Home,' 'Add Item,' and 'List' are concealed. Instead, they encounter a welcoming message and the Smart Shopping List logo. Additionally, the user is presented with two styled buttons, allowing them to choose between creating a new list or joining an existing one based on their preference. When the user types the name of an existing list, then the Alert is displayed.

## Note

Please note that the header style has already been updated in the open PR #35. 
As a team, we need to agree on whether we want to take the following actions for the Home Page: hide the links, keep them inactive, or completely hide the new navbar."

## Related Issue

closes#16

## Acceptance Criteria

- [x] The header should be hidden if the user is not logged into a list
- [x] The Home page should be styled  

## Type of Changes


|     | Type                       |
| --- | -------------------------- |
| ✓ | Add styling              |

## Updates

### Before
<img width="1423" alt="Screenshot 2023-10-03 at 7 45 25 PM" src="https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/49603115/adc24dea-41e3-43ec-a265-3af02e5ba305">

## After
<img width="1439" alt="Screenshot 2023-10-03 at 7 45 41 PM" src="https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/49603115/3ec5e30b-1c78-49a2-83ca-80f858e7e40a">

<img width="1434" alt="Screenshot 2023-10-03 at 9 01 29 PM" src="https://github.com/the-collab-lab/tcl-62-smart-shopping-list/assets/49603115/62e4e393-5c19-4e12-b2cc-575e300dc394">


## Testing Steps / QA Criteria
1. Go to the main branch in your code and get all the latest changes by using the command `git pull`.
2.  Switch to the `home-page-styling` branch.
3. Start your development environment by running the command `npm start`.
4. Access your browser's developer tools, and go to the Local Storage section under the Application tab. Clear the token stored there.
5. Refresh your website, and you should now observe the home page.
